### PR TITLE
optional argument for customize vm settings during creation

### PR DESCRIPTION
**Pull request by zakiva**

### CODE REVIEW

**Comment by simon3z** [comment]
@zakiva the tests are failing because you need to specify the new yaml requirement in the tox file.

File: virtdeploy/cli.py [hunk 1]
```diff
@@ -40,7 +41,8 @@
 
 def instance_create(args):
     driver = virtdeploy.get_driver(DRIVER)
-    instance = driver.instance_create(args.id, args.template)
+    args.settings = yaml.load(open(args.settings, 'r')) if (args.settings is not None) else {}
```
Comment (simon3z) [inline virtdeploy/cli.py]: Don't override variables. This would do:

```
settings = yaml.load(open(args.settings, 'r')) if args.settings is not None else {}
```

File: virtdeploy/cli.py [hunk 2]
```diff
@@ -124,6 +126,7 @@ def parse_command_line(cmdline):
     cmd_create = cmd.add_parser('create', help='create a new instance')
     cmd_create.add_argument('id', help='new instance id')
     cmd_create.add_argument('template', help='template id')
+    cmd_create.add_argument('settings', nargs='?', help='settings id')
```
Comment (simon3z) [inline virtdeploy/cli.py]: It's not `'settings id'`, it's `'settings yaml file'`

File: tox.ini [hunk 3]
```diff
@@ -1,12 +1,13 @@
 [tox]
 minversion = 1.6
-envlist = py27,py34,pep8,coverage
+envlist = py27,py3,pep8,coverage
```
Comment (simon3z) [inline tox.ini]: This should be in its own separate patch.

**Comment by zakiva** [comment]
@simon3z  please review
